tests/features/demo/demo_june.feature

```diff
@@ -157,6 +164,7 @@ Feature: June 2016 demo
       | Headline | New York under attack                                                                    |
       | Kicker   | S.H.I.E.L.D. to nuke New York?                                                           |
       | Content  | In a desperate attempt to stop the nuke, Nick fury shot down an airplane of S.H.I.E.L.D. |
+    And I select "Validated" from "State"
```

Comment:
Why is this changed in scope of this ticket? This is not about workflow. I guess this was working before?